src/layer/vector/Polyline.js

```diff
@@ -172,6 +172,21 @@ export var Polyline = Path.extend({
 		return this.redraw();
 	},
 
+	// @method setLastLatLng(latlng: LatLng, latlngs? LatLng[]): this
+	// Modifies the last point of the polyline to be the specified point.
+	// By default, adds to the first ring of the polyline in case of a
+	// multi-polyline, but can be overridden by passing a specific ring as a
```

Comment:
This is confusing. Multi-polylines don't have rings, because only polygons (and multi-polygons) have rings.

Reply:
This comment is similarly present in the `addLatLng()`, so I would think that any problem with this comment here should also apply to `addLatLng()` (just the previous method in this class)